README.md

```diff
@@ -1,148 +1,109 @@
 ```
-## State
-State describes the data provided by the user which the GUI will modify. VIZIA is reactive, which means that changes to the state will update the views.
- 
-### Defining State
-State is defined as a struct/enum which implements the `Model` trait:
-```rust
-#[derive(Lens)]
-struct AppData {
-	count: i32,
-}
 
-impl Model for AppData {}
+## Features
+ - Multiplatform (Windows, Linux, MacOS, Web)
+ - Declarative API
+ - Reactive with an elm-like archtecture
+ - Flexible layout engine, powered by [morphorm](https://github.com/vizia/morphorm)
+ - CSS-like styling with hot reloading
+ - Style property animations
 
-```
-The `Lens` derive macro enables binding views to fields of some state.
+## Learning Vizia
+<!-- Add docs to this when ready -->
+<div align="center">
+  <h3>
+    <a href="https://book.vizia.dev/"> Getting Started </a>
+    <span> | </span>
+    <a href="https://book.vizia.dev/"> The Book </a>
+    <span> | </span>
+    <a href="https://demo.vizia.dev/"> Web Demo </a>
+  </h3>
+</div>
 
-### Binding to State
-To use the state we need to `build` it into the view tree and then `bind` to it:
-```rust
-Application::new(window_description, |cx|{
-	
-	// Build the state into the tree
-	AppData{
-		count: 0,
-	}.build(cx);
-
-	HStack::new(cx, |cx|{
-		// Bind to the state
-		Binding::new(cx, AppData::count, |cx, some_data|{
-			Label::new(cx, *count.get(cx));
-		});
-	});
-}).run();
-```
-Anything within the body of the binding view will be updated when the bound data changes.
 
-### Mutating State
-To keep the separation of data changes and updates to the view tree, mutations of the data are done through events:
-```rust
-pub enum AppEvent {
-	Increment,
-	Decrement,
-}
+## Example Projects
 
-impl Model for AppData {
-	fn event(&mut self, cx: &mut Context, event: &mut MyEvent) {
-		if let Some(app_event) = event.message.downcast() {
-			match app_event {
-				AppEvent::Increment => self.count += 1,	
-				AppEvent::Decrement => self.count -= 1,
-			}
-		}
-	}
-}
-```
+Example projects go here... with pictures and stuff...
 
-### Events and Callbacks
-Some views have a built-in action or can be modified to add an action. Actions are callbacks which can be used to send events:
-```rust
-Application::new(WindowDescription::new(), |cx|{
-	
-	AppData{
-		count: 0,
-	}.build(cx);
-
-	HStack::new(cx, |cx|{
-		// Buttons take an action callback and a label
-		Button::new(cx, |cx| cx.emit(AppEvent::Increment), |cx|{
-			Label::new(cx, "Increment")
-		});
-
-		// The action will be called when the button is pressed
-		Button::new(cx, |cx| cx.emit(AppEvent::Decrement), |cx|{
-			Label::new(cx, "Decrement")
-		});
-
-		Binding::new(cx, AppData::count, |cx, some_data|{
-			Label::new(cx, &count.get(cx).to_string());
-		});
-	});
-}).run();
-```
+Vizia is also being used for the [Meadowlark project](https://github.com/MeadowlarkDAW/Meadowlark). An attempt to build a modern Digital Audio Workstation (DAW) with Rust.
+
+## Architecture
```

Comment:
Suggested text:

```
Vizia's core abstraction is a tree of **entities**. Each entity is just an integer, but this integer keys a number of dictionaries affecting how the entities look and act:
- **Style properties**, directives for layout and drawing
- **Models**, custom user-stored data which store a list of observers
- **Views**, bundles of callback logic to handle custom drawing, data reactivity, and user interaction

In order to interact with this tree of entities, there are several tools. The important ones are:
- **Bindings**, special views which watch for data changes and refresh their children
- **Events**, messages that propagate up or down the tree and trigger callbacks

To learn more, read the book, check out the examples, or join our discord!
```

Reply:
I like this very much, thank you!

Reply:
I've decided to remove the architecture section and will instead dedicate a whole file to it (eventually).